app/cdap/components/PipelineDiff/DiffIcon.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+import EditIcon from '@material-ui/icons/Edit';
+import AddBoxOutlinedIcon from '@material-ui/icons/AddBoxOutlined';
+import IndeterminateCheckBoxOutlinedIcon from '@material-ui/icons/IndeterminateCheckBoxOutlined';
+import { DiffIndicator } from './types';
+import { pluginDiffColors } from './util/helpers';
+
+interface IDiffIconProps {
+  diffType: DiffIndicator;
+  fontSize?: 'default' | 'inherit' | 'large' | 'medium' | 'small';
+}
+export function DiffIcon({ diffType, fontSize }: IDiffIconProps) {
+  if (diffType === '+') {
+    return (
+      <AddBoxOutlinedIcon
+        style={{ color: pluginDiffColors[diffType] }}
+        fontSize={fontSize ?? 'default'}
```

Comment:
'default' is a fontsize for material ui?

Reply:
Updated